apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.ts

```diff
@@ -493,95 +548,100 @@ function moveRowDown(event) {
   updateRubricItemOrderField();
 }
 
-function moveRowUp(event) {
+function moveRowUp(event: Event) {
   if (!(event.target instanceof HTMLElement)) return;
   const row = event.target.closest('tr');
   if (!row || !row.parentNode || !row.nextElementSibling || !row.previousElementSibling) return;
   row.parentNode.insertBefore(row.previousElementSibling, row.nextElementSibling);
   updateRubricItemOrderField();
 }
 
-function deleteRow(event) {
+function deleteRow(event: Event) {
   if (!(event.target instanceof HTMLElement)) return;
   const table = event.target.closest('table');
   event.target.closest('tr')?.remove();
   if (!table?.querySelectorAll('.js-rubric-item-row-order')?.length) {
-    table.querySelector('.js-no-rubric-item-note')?.classList.remove('d-none');
+    table?.querySelector('.js-no-rubric-item-note')?.classList.remove('d-none');
   }
   updateRubricItemOrderField();
   checkRubricItemTotals();
 }
 
-function rowDragStart(event) {
+function rowDragStart(event: JQuery.TriggeredEvent) {
   if (!(event.target instanceof HTMLElement)) return;
-  window.rubricItemRowDragging = event.target.closest('tr');
-  if (event.originalEvent?.dataTransfer) {
+  const closestRow = event.target.closest('tr');
+  if (closestRow) {
+    window.rubricItemRowDragging = closestRow;
+  }
+
+  if (!(event.originalEvent instanceof DragEvent)) return;
```

Comment:
`event.originalEvent` is jQuery specific; which causes a type mismatch at the caller location. My recommendation would be to either use a jquery-specific event listener, or rewrite this code to avoid this issue.